prow/pod-utils/gcs/metadata.go

```diff
@@ -42,8 +41,8 @@ type Finished = metadata.Finished
 //
 // and the simplified filename would be build-log.txt (excluding the
 // content encoding extension).
-func AttributesFromFileName(filename string) (string, *storage.ObjectAttrs) {
```

Comment:
Why are we confident that there are no unintended systemic side effects switching `storage.ObjectAttrs` to `blob.Attributes`?